### DISCUSSION TITLE
AuditLog Implementation in Asp.net Core

### DISCUSSION THREAD

**Discussion by gfo2007** [serenity-is/Serenity#5948] [Help]
Hi,
I am trying to implement AuditLog as demonstrated in the wiki.

However, I get a lot of compile time errors. The image below shows some of the Errors.

Is this implementation not applicable in serenity asp.net core apps? If not, is there any other implementation for Logging all inserts, updates and deletes into one sql server table?

![AuditLog error](https://user-images.githubusercontent.com/14317420/131247831-c1102ddb-8e5f-4d30-8491-bb3ef2315619.PNG)

Regards,

**Comment by edwardch** [comment]
Hey @gfo2007, that wiki entry has been around since before .NET 5 or Serenity v5.x, so maybe a good idea to refer to the [Upgrading to Serenity.NET 5](https://serenity.is/docs/migration/v3-to-v5) doc to see how to fix those errors. 

If you get it working it would be great to have some contribution/update to the wiki!
